src/Symfony/Bundle/FrameworkBundle/Resources/config/form_csrf.xml

```diff
@@ -12,6 +12,7 @@
             <argument>%form.type_extension.csrf.field_name%</argument>
             <argument type="service" id="translator.default" />
             <argument>%validator.translation_domain%</argument>
+            <deprecated>"The \"%service_id%\" service is deprecated since Symfony 3.1 and will be removed in 4.0."</deprecated>
```

Comment:
@stof I was wondering this too, it's actually returned in `CsrfExtension::loadTypeExtensions()`, the same goes for the other cases. I suppose those classes are used as factory for this instead of the services here.